### DISCUSSION TITLE
Problem with custom actions

### DISCUSSION THREAD

**Discussion by Ju-80** [helgoboss/helgobox#1263] [Q&A]
Hello, I've a problem with realearn : I have a reaper project in which I use a realearn instance with my X-touch controller. Evrything works fine except one thing :

Regularly realearn change all the custom actions that I assigned to buttons.
He change only the custom actions, never the native actions or the others realearn partameters like track selection, track volume etc.

It happens every time that reaper ask me to restart after installing or updating a extension.

Today it happens yet, and the difference is that I don't install or update any extension, but yesterday I ad a new realearn instance in the same project, for another controler, and I create new custom actions.

You can see in the pictures an exemple of mapping that I made, and after the mapping that change by theyselve.

![Realearn normal 2](https://github.com/user-attachments/assets/97ae2896-9fb9-4c58-80c1-f3c6bcd3c77f)
![Realearn problem 2](https://github.com/user-attachments/assets/eb688a93-8252-419f-8b08-9a0e4df902a5)

**Comment by helgoboss** [comment]
Hi. You discovered a critical bug. See #1272. Will discuss and fix it there.
